exporter/views.py

```diff
@@ -8,57 +7,48 @@
 from django.http import HttpResponse
 from django.utils.decorators import method_decorator
 from django.views.generic import TemplateView
+from tabular_export.core import export_to_csv_response, flatten_queryset
 
-from concordia.models import Asset, Campaign, Transcription, UserAssetTagCollection
+from concordia.models import Asset, Campaign, Transcription
 from concordia.storage import ASSET_STORAGE
 
 
 class ExportCampaignToCSV(TemplateView):
     """
-    Exports the transcription and tags to csv file
+    Exports the most recent transcription for each asset in a campaign
 
     """
 
-    template_name = "transcriptions/campaign.html"
-
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
-        campaign = Campaign.objects.get(slug=self.kwargs["campaign_slug"])
-        asset_list = Asset.objects.filter(item__project__campaign=campaign).order_by(
-            "title", "sequence"
-        )
-        # Create the HttpResponse object with the appropriate CSV header.
-        response = HttpResponse(content_type="text/csv")
-        response["Content-Disposition"] = 'attachment; filename="{0}.csv"'.format(
-            campaign.slug
+        qs = Campaign.objects.filter(slug=self.kwargs["campaign_slug"]).select_related()
```

Comment:
I generally like to start with the class I'm returning (e.g. `Asset` in this case) but I don't think that matters here since we're returning the whole table anyway.

Reply:
Don't we need to filter for the most recent transcription so we don't return multiple rows for every revision? Since we're using Django 2, I think that's just a [`Subquery`](https://docs.djangoproject.com/en/2.1/ref/models/expressions/#subquery-expressions) annotation.

Reply:
Yes, I was trying to figure out how to do that